backend/models/helper/model.js

```diff
@@ -288,17 +268,15 @@ function createNewModel(teamspace, modelName, data) {
 			return Promise.reject(responseCodes.PROJECT_NOT_FOUND);
 		}
 
+		// FIXME when project changes are merged, consider using func in project
```

Comment:
I was mistaken and no such func exists in project to check model names. Also, since this function is checking the "settings" collection, maybe it's better to keep this in model settings.